tf2_eigen/CMakeLists.txt

```diff
@@ -1,29 +1,19 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_eigen)
 
-find_package(catkin REQUIRED COMPONENTS
-  cmake_modules
-  geometry_msgs
-  tf2
-)
+find_package(ament_cmake REQUIRED)
 
-find_package(Eigen REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(tf2 REQUIRED)
+
+find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN_INCLUDE_DIRS})
 add_definitions(${EIGEN_DEFINITIONS})
```

Comment:
I have no clue.  I didn't even consider Windows right now.  One thing I could do would be to do a "soft" lookup of Eigen3, and then fall back to finding Eigen if that fails (I do this in some of my ROS1 packages).  Do you think that would make it work better on Windows?